tests/ast.test.ts

```diff
@@ -37,8 +52,139 @@ describe('ast', () => {
                 }
             }
         };
-        const node = asNode(file.parseMotokoTyped().ast);
+        const node = asNode(file.parseMotokoTyped(new Map<string, Scope>())[0].ast);
```

Comment:
We could test backwards compatibility here:

```suggestion
        const node = asNode(file.parseMotokoTyped().ast);
```